### PR TITLE
BaSP-TG-14: feature/TG-14/test-projects-Luciano

### PR DESCRIPTION
**Pull request by Clarosluciano**
- [x]  Tested the GET request.
      *  Tested if the project is correctly getted
      *  Tested if it returns a false error
      *  Tested if the response is 200
      *  Tested if the response is 404
      *  Tested if all keys length is greater than 0
- [x]  Tested the PUT request.
      *  Tested if all keys length is greater than 0
      *  Tested if the project is correctly putted
      * Tested if the response is 404
      *  Tested if the response is 200
      *  Tested if the response is 400

### CODE REVIEW

**Review by Julian-Cano — COMMENTED** [review]
Luciano, I don't understand why you deleted the employee validations inside create and update project validations

File: src/models/Projects.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ const projectSchema = new Schema(
...
+    status: { type: Boolean, required: true },
     client: { type: String, required: true },
     // startDate: { type: Date, required: true },
     // endDate: { type: Date, required: true },
     employees: [
       {
         type: mongoose.Schema.Types.ObjectId,
-        required: true,
-        ref: 'Employees',
+        require: true,
```
Comment (GuidoCerioni) [inline src/models/Projects.js:16]: This key must be "required"

**Review by GuidoCerioni — COMMENTED** [review]
Watch out the line i comment in project model

File: src/models/Projects.js (hunk 1 above)
Comment (Clarosluciano) [reply src/models/Projects.js]: Change done, I think so

**Review by Julian-Cano — APPROVED** [review]
Good work Luciano. Now I understand why you removed the employees validation inside project creation validation.

**Review by AgustinCarthery — APPROVED** [review]

**Pull request merged**